src/test/java/teammates/test/cases/util/SanitizationHelperTest.java

```diff
@@ -358,4 +358,19 @@ public void testDesanitizeIfSanitized() {
         assertEquals("should return same unsanitized string if given unsanitized string containing sanitized sequences",
                 unsanitized, SanitizationHelper.desanitizeIfHtmlSanitized(unsanitized));
     }
+
+    @Test
+    public void testSanitizeForLogMessage() {
+        assertNull("should return null if given null", SanitizationHelper.sanitizeForLogMessage(null));
+
+        String unsanitized = "<span class=\"text-danger\"> A <span class=\"bold\">typical</span> log  message <br>"
```

Comment:
Some HTML elements with other class?

Reply:
currently i'm only allowing `bold` and `text-danger` as that's what i found the log messages to be using. Other classes will be escaped.

Reply:
That is exactly I am asking for, to test that scenario also and make sure it works